projects/igniteui-angular/src/lib/data-operations/data-clone-strategy.ts

```diff
@@ -9,3 +9,9 @@ export class DefaultDataCloneStrategy implements IDataCloneStrategy {
         return cloneValue(data);
     }
 }
+
+export class CachedDataCloneStrategy implements IDataCloneStrategy {
```

Comment:
This file is exported to the public API, so unless we want to make this public you might want to set hidden/internal on it and modify the API exports to exclude it. Perhaps it can be useful as public, though the issue it solves is a particularly narrow and not the goal of this PR anyway, so I'd go with hiding it.
I'd also throw a hidden on the `cloneValueCached` for good measure